packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/advanced-features/advanced-features.component.js

```diff
@@ -0,0 +1,19 @@
+import controller from './advanced-features.controller';
+import template from './advanced-features.html';
+
+export default {
+  bindings: {
+    /**
+     * {Object}     sgx
+     * {boolean}    sgx.isRunning - is the status running or completed ?
+     * {string}     sgx.prmrr - the RAM memory reserved for SGX
+     * {SgxStatus}  sgx.status - the status of the SGX option
```

Comment:
Maybe `sgx` should be an instance of `SGX` defined in a `SGX.class.js` then

Reply:
I don't know, aren't classes in the models only data from the API ? Because `sgx` is mostly custom made

What make the more sense is having a `sgx.class.js` and having a `sgx-dashboard.class.js`  , what do you think ?